src/app/modules/main/wallet_section/activity/entry.nim

```diff
@@ -264,51 +274,31 @@ QtObject:
   QtProperty[string] nonce:
     read = getNonce
 
-# TODO: Replaced usage of these for in/out versions in the QML modules
-  proc getSymbol*(self: ActivityEntry): string {.slot.} =
+  proc getBlockNumber*(self: ActivityEntry): string {.slot.} =
     if self.transaction == nil:
-      error "getSymbol: ActivityEntry is not an transaction.Item"
+      error "getNonce: ActivityEntry is not an transaction.Item"
       return ""
-    
-    if self.activityType == backend.ActivityType.Receive:
-      return self.getInSymbol()
-
-    return self.getOutSymbol()
-
-  QtProperty[string] symbol:
-    read = getSymbol
+    return $self.transaction[].getBlockNumber()
 
-  proc getFromAmount*(self: ActivityEntry): float {.slot.} =
-    if self.isMultiTransaction():
-      return self.getOutAmount()
-    error "getFromAmount: ActivityEntry is not a MultiTransaction"
-    return 0.0
-
-  QtProperty[float] fromAmount:
-    read = getFromAmount
+  QtProperty[string] blockNumber:
+    read = getBlockNumber
 
-  proc getToAmount*(self: ActivityEntry): float {.slot.} =
-    if self.isMultiTransaction():
-      return self.getInAmount()
-    error "getToAmount: ActivityEntry is not a MultiTransaction"
-    return 0.0
+  proc getOutValue*(self: ActivityEntry): QVariant {.slot.} =
+    return newQVariant(self.extradata.outValue)
 
-  QtProperty[float] toAmount:
-    read = getToAmount
+  QtProperty[QVariant] outValue:
+    read = getOutValue
 
-  proc getValue*(self: ActivityEntry): float {.slot.} =
-    if self.isMultiTransaction():
-      error "getToAmount: ActivityEntry is a MultiTransaction"
-      return 0.0
+  proc getInValue*(self: ActivityEntry): QVariant {.slot.} =
+    return newQVariant(self.extradata.inValue)
 
-    if self.activityType == backend.ActivityType.Receive:
-      return self.getInAmount()
+  QtProperty[QVariant] inValue:
+    read = getInValue
 
-    # For some reason status-go is categorizing every activity as Receive,
```

Comment:
Hmm you sure this workaround is not needed anymore? I can still reproduce the issue on my side